domains/quantummapleqc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "repo": "https://github.com/QuantumMapleQC/quantumapleqc.github.io",
+  "owner": {
+    "username": "quantumapleqc",
+    "email": "thefattestmod@gmail.com",
+  },
+  "record": {
+    "CNAME": "quantummapleqc.github.io"
```

Comment:
404 CNAME page, please have actual content.

Reply:
the issue is that my page is double github domain for some reason 

heres the link https://quantummapleqc.github.io/quantumapleqc.github.io/